source/command/atlas-api.txt

```diff
@@ -0,0 +1,154 @@
+.. _atlas-api:
+
+=========
+atlas api
+=========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Access all features of the `{+atlas-admin-api+} <https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/>`__ 
+using the {+atlas-cli+} with the syntax: ``atlas api <tag> <operationId>``.
+
+This experimental feature streamlines script development by letting you interact 
+directly with any {+atlas-admin-api+} endpoint using the {+atlas-cli+}.
+
+To learn more about working with the {+atlas-admin-api+}, see :ref:`atlas-admin-api-access`.
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas api <tag> <operationId> [options]
```

Comment:
While it's correct that we use `<tag>` and `<operationId>`, we use a `camelCase` version of these values.

This will mainly influence `<tag>`, afaik the current operationIds are already in camelCase.

Some examples:
- `Flex Clusters` -> `flexClusters`
- `Monitoring and Logs` -> `monitoringAndLogs`

I don't see this captured anywhere in this page yet.